Tests/Common/SonarQube.Plugins.Test.Common.csproj

```diff
@@ -5,6 +5,7 @@
         <PackageReference Include="MSTest.TestAdapter" Version="3.3.1" />
         <PackageReference Include="MSTest.TestFramework" Version="3.3.1" />
         <PackageReference Include="FluentAssertions" Version="6.12.0" />
+        <PackageReference Include="System.IO.Compression" Version="4.3.0" />
```

Comment:
For some reason, I still to at least explicit specify the dependency:
```xml
<Reference Include="System.IO.Compression" />
```

Reply:
Hmm, my 'build' was probably not enough to clear up all the cache. Rebuild finally failed, so we need the line.
`Reference` takes it from GAC, because we're in net48 TFM. There's less to maintain, as it doesn't contain the verison. 
`PackageReference` takes it from NuGet. It's nicer in the code, as it's more regular.

I don't have a strong argument for one over another.